zkevm-circuits/src/evm_circuit/execution/push.rs

```diff
@@ -35,10 +40,33 @@ impl<F: Field> ExecutionGadget<F> for PushGadget<F> {
 
     fn configure(cb: &mut EVMConstraintBuilder<F>) -> Self {
         let opcode = cb.query_cell();
+        let code_size = cb.query_cell();
+        let code_size_left = code_size.expr() - cb.curr.state.program_counter.expr();
```

Comment:
In the specs this is `code_length_left = code_length - instruction.curr.program_counter - 1`.  I wonder why the `-1` is not here.

Imagine the `code_size=100`.  And where at `program_counter=99` (the last byte in the bytecode).  Then `code_size_left` should be `0`.

Reply:
I am not sure why I did not write or removed the -1, sorry about that!